README.md

```diff
@@ -1 +1,11 @@
-# fibonacci
\ No newline at end of file
+# Fibonacci
+Compute the nth fibonacci number.
```

Comment:
It'd probably be helpful to link to the [Wikipedia article for Fibonacci number](https://en.wikipedia.org/wiki/Fibonacci_number).

Reply:
done!